css/builder.css

```diff
@@ -551,7 +551,7 @@ input[type="time"].form-control {
 }
 
 input[type="time"]::-webkit-inner-spin-button,
-input[type="time"]::-webkit-calendar-picker-indicator {
+input[type="time"]:not(.basic-time-picker)::-webkit-calendar-picker-indicator {
```

Comment:
What's the consequence if we didn't do this?

Reply:
@tonytlwu if don't make this changes this Clock icon is not getting displayed.  @NikitaJadhavXoriant  correct me if I am wrong

Reply:
@yogeshmsharma-xoriant, yes agreed, as the basic time picker icon ( -webkit-calendar-picker-indicator ) is hidden for custom design, it will not show and user will not understand where to click to open timer picker.